### ISSUE TITLE
Feature Request: Change Origin

### ISSUE THREAD

**Issue by DylanLogan2581** [SamScherf/biras#1]
In order to add on to an already existing map, can you please implement a way to change the origin coordinates (the top left output image) from r.0.0 to something arbitrary so that a second image can be cut up and arranged alongside an existing map?

**Comment by SamScherf** [comment]
So this feature would only change the output image's name?

**Comment by DylanLogan2581** [comment]
Yes. Just to rename the top left images coordinates, and then all of the others accordingly.
